Classes/Events/BeforeContainerPreviewIsRenderedEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace B13\Container\Events;
+
+/*
+ * This file is part of TYPO3 CMS-based extension "container" by b13.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ */
+
+use B13\Container\Domain\Model\Container;
+use TYPO3\CMS\Fluid\View\StandaloneView;
+
+final class BeforeContainerPreviewIsRenderedEvent
+{
+    protected Container $container;
+
+    protected StandaloneView $view;
```

Comment:
I think we need to check w @lolli42 how to handle this properly due to the discontinuation of StandaloneView in v13